### PR TITLE
Tasks/serializer fixes

### PR DESCRIPTION
**Pull request by alashow**

### CODE REVIEW

File: apps/moderation/tests/test_serializers.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+from .moderated_serializers import MODERATED_SERIALIZERS as serializers
+
+
+@pytest.mark.django_db()
+class TestModeratedSerializers:
+
+    # allow these fields to have choices instead of instant search
+    instant_search_field_exceptions = ['birth', 'death']
+
+    @classmethod
```
Comment (iacobfred) [inline apps/moderation/tests/test_serializers.py:16]: Why class methods?

File: apps/search/documents/entity.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from apps.entities.models.entity import Entity
 from apps.search.documents.config import DEFAULT_INDEX_SETTINGS, html_field_analyzer
 
+from ...entities.models import Category
```
Comment (iacobfred) [inline apps/search/documents/entity.py:7]: nit: prefer absolute imports like `from apps.entities.models import Category`

File: apps/moderation/tests/test_serializers.py (hunk 1 above)
Comment (alashow) [reply apps/moderation/tests/test_serializers.py:16]: IDE kept saying it should be classmethod since they're not using anything self. Should I change?

Comment (iacobfred) [reply apps/moderation/tests/test_serializers.py:16]: Gotcha. No need to change it )

File: apps/moderation/serializers.py [hunk 3]
```diff
@@ -46,13 +59,14 @@ def get_moderated_fields(self) -> list[dict]:
...
                     'type': field.__class__.__name__,
-                    'editable': not getattr(field, 'read_only', False),
-                    'required': not getattr(field, 'required', False),
-                    'allow_blank': not getattr(field, 'allow_blank', False),
+                    'editable': True,  # TODO: remove after front-end code is updated
+                    'required': getattr(field, 'required', False),
+                    'allow_blank': getattr(field, 'allow_blank', False),
                     'verbose_name': getattr(field, 'verbose_name', None),
                     'help_text': getattr(field, 'help_text', None),
                     'choices': self.get_choices_for_field(field, field_name),
```
Comment (mattfredericksen) [inline apps/moderation/serializers.py:72]: Why do we want `editable: True` here?

**Review by mattfredericksen — APPROVED** [review]
LGTM

**Comment by alashow** [comment]
I assumed frontend might be using it?

**Comment by mattfredericksen** [comment]
> I assumed frontend might be using it?

Oh, I meant why `editable: True` instead of `not getattr(field, 'read_only', False)`.

**Comment by alashow** [comment]
> > I assumed frontend might be using it?
> 
> Oh, I meant why `editable: True` instead of `not getattr(field, 'read_only', False)`.

Because now we're filtering out non editable fields (with `if is_editable`)

**Pull request merged**